src/azure-cli/azure/cli/command_modules/sql/_format.py

```diff
@@ -244,16 +244,15 @@ def elastic_pool_transform(result):
     '''
     Transforms the json response for an elastic pool.
     '''
-    from azure.mgmt.sql.models import ElasticPoolEdition
 
     # Add properties in order to improve backwards compatibility with api-version 2014-04-01
     result.edition = result.sku.tier
     result.storageMb = result.max_size_bytes / 1024 / 1024
 
     is_dtu = result.sku.tier in (
-        ElasticPoolEdition.basic.value,
-        ElasticPoolEdition.standard.value,
-        ElasticPoolEdition.premium.value)
+        'Basic',
+        'Standard',
+        'Premium')
```

Comment:
will these values change in the future?

Reply:
No

Reply:
no

Reply:
They deprecated the boolean so we had to switch to actual values in the version bump

Reply:
I posted on the chat group to get answers from the feature owner

Reply:
Feature owner is oof till 7/12, we will miss our enterprise promise deadlines if we wait for his answer